### ISSUE TITLE
Permission denied

### ISSUE THREAD

**Issue by rjaffeco1** [docker/for-linux#1502]
I am attempting to execute the following docker load command on a host computer with ubuntu 18.04 OS installed.

I entered the following command on terminal session:
 
magic@magic-OptiPlex-5250-AIO:~/Downloads$ docker load -i ./sdkmanager-2.2.0.12021-Ubuntu_18.04_docker.tar.gz

The following results were displayed:
 
permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/images/load?quiet=0": dial unix /var/run/docker.sock: connect: permission denied
magic@magic-OptiPlex-5250-AIO:~/Downloads$ 

Do I need to install docker desktop on my host computer. Please walk me thru the steps

Thanks for your support 

warmly, bob

**Comment by IsusRamzy** [comment]
`sudo`

**Comment by thaJeztah** [comment]
Yes, this is expected for a default Docker Engine on Linux setup. Having access to the Docker Engine API is equivalent to `root` access on the host. Because of that the default permissions on the socket require you to either be `root` (or using `sudo`), or for the user to be part of the `docker` group; https://docs.docker.com/engine/install/linux-postinstall/

You can use Docker Desktop for Linux (https://docs.docker.com/desktop/setup/install/linux/), which runs the daemon inside a non-privileged VM, or run docker in rootless mode (https://docs.docker.com/engine/security/rootless/)


Let me close this ticket as it's not a bug, but feel free to continue the converrsatoni
